Automattic/wp-calypso
# Fix: Remove Jetpack Social Basic from Complete plan

As mentioned here: p1676949662395079-slack-C01264051NE

On the pricing page, Jetpack Complete currently includes both Jetpack Social Basic and Jetpack Social Advanced. Since Advanced contains all features of Basic, there's no need to include Basic separately.

## Proposed Changes

* Remove Jetpack Social Basic from the Jetpack Complete plan on the pricing page.

## Testing Instructions



* Open a live link below in Jetpack Cloud.
* Visit the Pricing Page, find the Jetpack Complete plan and click "More about Complete".
* Ensure Social is only included in the products list once.

## Pre-merge Checklist
